microsoft-onedrive/1.0/modules/ROOT/pages/microsoft-onedrive-connector-reference.adoc

```diff
@@ -0,0 +1,1317 @@
+= Microsoft OneDrive Module Connector Reference
```

Comment:
DriveItem in the descriptive text should really be "drive item," but it's up to you whether to expend the effort on this.

Reply:
It's actually called a driveItem in the Microsoft API documentation.